source/common/upstream/subset_lb.cc

```diff
@@ -186,34 +205,18 @@ SubsetLoadBalancer::LbSubsetEntryPtr SubsetLoadBalancer::findSubset(
 
 void SubsetLoadBalancer::updateFallbackSubset(uint32_t priority, const HostVector& hosts_added,
```

Comment:
update name? as this method now updates not only default subset

Reply:
hmm the panic mode subset is tied to the fallback subset.. not sure what a better name would be.